README.md

```diff
@@ -11,4 +11,5 @@ ClickHouse® is an open-source column-oriented database management system that a
 * [Slack](https://join.slack.com/t/clickhousedb/shared_invite/zt-rxm3rdrk-lIUmhLC3V8WTaL0TGxsOmg) and [Telegram](https://telegram.me/clickhouse_en) allow chatting with ClickHouse users in real-time.
 * [Blog](https://clickhouse.com/blog/en/) contains various ClickHouse-related articles, as well as announcements and reports about events.
 * [Code Browser](https://clickhouse.com/codebrowser/html_report/ClickHouse/index.html) with syntax highlight and navigation.
+* [Code Browser@github.dev](https://github.dev/ClickHouse/ClickHouse) with syntax highlight and navigation, powered by github.dev free of charge.
```

Comment:
C/C++ extension is not available in the Web version of VS Code so syntax highlight with navigation do not work properly :(